doc/userguide/rules/dns-keywords.rst

```diff
@@ -1,10 +1,28 @@
 DNS Keywords
 ============
 
-There are some more content modifiers (If you are unfamiliar with
-content modifiers, please visit the page :doc:`payload-keywords` These
-ones make sure the signature checks a specific part of the
-network-traffic.
+Suricata supports sticky buffers as well as keywords for efficiently
+matching on specific fields in DNS messages.
+
+Note that sticky buffers are expected to be followed by one or more
+:doc:`payload-keywords`.
+
+dns.answer.name
+---------------
+
+``dns.answer.name`` is a sticky buffer that is used to look at the
+name field in DNS answer resource records.
+
+``dns.answer.name`` will look at both requests and responses, so
+``flow`` is recommended to confine to a specific direction.
+
+.. note:: At this time ``dns.answer.name`` is only supported for
+          responses, but supporting it in requests is planned.
```

Comment:
These two portions felt a bit ambiguous to me. Am I missing something?

Reply:
it will, but not yet.